libsplinter/src/network/peer_manager/connector.rs

```diff
@@ -97,6 +97,37 @@ impl PeerManagerConnector {
             .map_err(|err| PeerRefAddError::ReceiveError(format!("{:?}", err)))?
     }
 
+    /// Request that a peer is added to the PeerManager. If a peer already exists, the peer's ref
+    /// count will be incremented. This function should be used when the peer id is unknown.
+    ///
+    /// # Arguments
+    ///
+    /// * `endpoint` -  The  endpoints associated with the peer.
```

Comment:
Should this be "endpoint" or "endpoints"? The description of the argument says "endpoints" (also looks like a double space here)

Reply:
should be endpoint, fixed